recipes/easydict/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.9" %}
+package:
+    name: easydict
+    version: {{ version }}
+source:
+    sha256: 3f3f0dab07c299f0f4df032db1f388d985bb57fa4c5be30acd25c5f9a516883b
+    url: https://pypi.io/packages/source/e/easydict/easydict-{{ version }}.tar.gz
+build:
+    number: '0'
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    host:
+        - python
+        - pip
+    run:
+        - python
+test:
+    imports:
+        - easydict
+about:
```

Comment:
You need to add a `license_file` entry to the about section with the name of the license file. This is required by the LGPL license.